Rakefile

```diff
@@ -2202,6 +2205,12 @@ def get_hpxml_file_water_heating_system_values(hpxml_file, water_heating_systems
   elsif ['base-dhw-uef.xml'].include? hpxml_file
     water_heating_systems_values[0][:energy_factor] = nil
     water_heating_systems_values[0][:uniform_energy_factor] = 0.93
+  elsif ['base-dhw-jacket-electric.xml'].include? hpxml_file
+    water_heating_systems_values[0][:jacket_r_value] = 10.0
+  elsif ['base-dhw-jacket-indirect.xml'].include? hpxml_file
+    water_heating_systems_values[0][:jacket_r_value] = 10.0
+  elsif ['base-dhw-jacket-gas.xml'].include? hpxml_file
+    water_heating_systems_values[0][:jacket_r_value] = 10.0
```

Comment:
Since all three elsif statements make the same change, you can combine with using:
```
elsif ['base-dhw-jacket-electric.xml',
       'base-dhw-jacket-indirect.xml',
       'base-dhw-jacket-gas.xml'].include? hpxml_file
```